src/rules/at-function-named-arguments/index.js

```diff
@@ -0,0 +1,133 @@
+import { utils } from "stylelint";
+import { namespace, optionsHaveIgnored } from "../../utils";
+import valueParser from "postcss-value-parser";
+
+export const ruleName = namespace("at-function-named-arguments");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "Expected a named parameter to be used in function call",
+  rejected: "Unexpected a named parameter in function call",
+  rejectedSingle:
+    "Unexpected a named parameter in single argument function call"
+});
+
+const cssFunctions = [
+  "attr",
+  "calc",
+  "linear-gradient",
+  "radial-gradient",
+  "repeating-linear-gradient",
+  "repeating-radial-gradient"
```

Comment:
Oh, btw. this function list seems incomplete, here's the full one:
https://github.com/EtaiG/css-functions-list

Reply:
and maybe you could also add a test that tests a CSS function that uses camelCase, for example `translateZ`.

Reply:
Good call!